mobilecoind-json/src/data_types.rs

```diff
@@ -1091,7 +1091,7 @@ impl From<&mc_mobilecoind_api::ProcessedTxOut> for JsonProcessedTxOut {
         Self {
             monitor_id: hex::encode(&src.get_monitor_id()),
             subaddress_index: src.subaddress_index,
-            public_key: hex::encode(&src.get_public_key().get_data()),
+            public_key: hex::encode(&src.get_tx_public_key().get_data()),
```

Comment:
This should actually be `get_tx_out_public_key` if we want to be specific. There's no public key for a transaction, only for a TxOut.